pertpy/tools/_coda/_base_coda.py

```diff
@@ -1535,6 +1537,8 @@ def plot_boxplots(  # pragma: no cover # noqa: D417
             args_swarmplot = {}
         if isinstance(data, MuData):
             data = data[modality_key]
+        if isinstance(palette, Colormap):
+            palette = np.array([palette(i % palette.N) for i in range(2)]).tolist()
```

Comment:
Not the most easy version and I wonder whether just

`palette = [palette(i) for i in range(2)]`

would be enough?

Reply:
I actually think if we assume `palette.N > 1`, `palette = palette(range(2))` should do.

Reply:
Okie! If you test that quickly and could push it, I'm happy to merge the PR.